rasa/train.py

```diff
@@ -125,38 +164,90 @@ async def handle_domain_if_not_exists(
     return nlu_model_only
 
 
+def dry_run_result(
+    fingerprint_comparison: FingerprintComparisonResult,
+) -> Tuple[int, List[Text]]:
+    """Returns a dry run result.
+
+    Args:
+        fingerprint_comparison: A result of fingerprint comparison operation.
+
+    Returns:
+        A tuple where the first element is the result code and the second
+        is the list of human-readable texts that need to be printed to the end user.
+    """
+    code = 0
+    texts = []
+
+    if fingerprint_comparison.core:
+        code += CODE_CORE_NEEDS_TO_BE_RETRAINED
+        texts.append("Core model should be retrained.")
+
+    if fingerprint_comparison.core:
+        code += CODE_NLU_NEEDS_TO_BE_RETRAINED
+        texts.append("NLU model should be retrained.")
+
+    if fingerprint_comparison.nlg:
+        code += CODE_NLG_NEEDS_TO_BE_RETRAINED
+        texts.append("Responses in the domain should be updated.")
+
+    if fingerprint_comparison.force_training:
+        code += CODE_FORCED_TRAINING
```

Comment:
doesn't make a big difference, but would that be safer?
```suggestion
        code = CODE_FORCED_TRAINING
```